src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
+  const newSeparator = toFormat.slice(3);
```

Comment:
The `slice` method is being used incorrectly to extract the separator from the `fromFormat` and `toFormat` arrays. Since the separator is the fourth element in these arrays (index 3), you should use indexing to access it directly, for example: `const oldSeparator = fromFormat[3];`